src/APIClient.php

```diff
@@ -48,4 +58,51 @@ private function buildRequest(CommandDTO $command, array $headers): RequestInter
 
         return $request;
     }
+
+    private function logRequest(RequestInterface $request): void
+    {
+        $this->logger->debug("Request to Admitad API is built (DEBUG)", [
+            'uri' => (string) $request->getUri(),
+            'method' => $request->getMethod(),
+            'headers' => $request->getHeaders(),
+            'body' => $request->getBody()->getContents(),
+        ]);
+        $request->getBody()->rewind();
+    }
+
+    private function logResponse(RequestInterface $request, ResponseInterface $response): void
+    {
+        $debugLevelRequestData = [
+            'uri' => (string) $request->getUri(),
+            'method' => $request->getMethod(),
+            'headers' => $request->getHeaders(),
+            'body' => $request->getBody()->getContents(),
+        ];
+
+        $debugLevelResponseData = [
+            'status' => $response->getStatusCode(),
+            'message' => $response->getReasonPhrase(),
+            'headers' => $response->getHeaders(),
+            'body' => json_decode($response->getBody()->getContents(), true),
+        ];
+        $this->logger->debug('Response from Admitad API (DEBUG)', [
+            'request' => $debugLevelRequestData,
+            'response' => $debugLevelResponseData,
+        ]);
+
+        $infoLevelRequestData = $debugLevelRequestData;
+        unset($infoLevelRequestData['headers']['Authorization']);
+
+        $infoLevelResponseData = $debugLevelResponseData;
+        unset($infoLevelResponseData['body']['access_token']);
```

Comment:
You should leave this keys within response but replace values of `access_token` and `response_token` with `***`.

Reply:
done